### DISCUSSION TITLE
GSOC 2022: Interested in "Simple Repository Hosting" project idea

### DISCUSSION THREAD

**Discussion by rishits321** [koordinates/kart#590] [Google Soc / SoD]
Hello all,
I am Rishikesh Suryavanshi and I am currently pursuing a bachelor's in Information Technology at Pune Institute of Computer Technology.
I have a firm grasp of c++, python, and git. Here are my google cloud [qwiklabs](https://www.qwiklabs.com/public_profiles/1706f4b9-fb4e-4be8-91cb-cbeec5a137fd) certifications: 

I would love to contribute to the kart organization for GSOC 2022. I am very excited to work on the project -"Simple Repository Hosting".
This project falls well in my current capabilities and I will be very excited to work on it. I also want to know the codebase and would like to start contributing right away.

@rcoup 
I look forward to hearing from you and further working on this project.

**Comment by rcoup** [comment]
Hi Rishikesh,

You'll need to do some work to understand the issues and how a solution can fit together. Essentially:
- understanding how gitolite works
- and how it could be built and sensibly configured with Kart (and Kart's Git distribution)
- and how maintenance tasks can be setup to repack/etc repositories
- and how openssh, gitolite, and Kart could be packaged together usefully into a Docker container
